src/main/java/ch/njol/skript/expressions/ExprLastDeathLocation.java

```diff
@@ -0,0 +1,57 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import org.bukkit.Location;
+import org.bukkit.entity.Player;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Last Death Location")
+@Description("The location of a player's last death.")
+@Examples("set {lastdeathlocation::%player's uuid%} to last death location of player")
+@Since("INSERT VERSION")
+public class ExprLastDeathLocation extends SimplePropertyExpression<Player, Location> {
+	
+	static {
+		register(ExprLastDeathLocation.class, Location.class, "last death location", "players");
```

Comment:
This method was added in 1.14. Needs a version check (not a method check as we know the version) as Skript supports 1.13.